great_expectations/datasource/data_connector/__init__.py

```diff
@@ -1,23 +1,37 @@
-from .configured_asset_file_path_data_connector import (
+from .data_connector import DataConnector  # isort:skip
+from .runtime_data_connector import RuntimeDataConnector  # isort:skip
+from .file_path_data_connector import FilePathDataConnector  # isort:skip
+from .configured_asset_file_path_data_connector import (  # isort:skip
     ConfiguredAssetFilePathDataConnector,
 )
-from .configured_asset_filesystem_data_connector import (
+from .inferred_asset_file_path_data_connector import (  # isort:skip
+    InferredAssetFilePathDataConnector,
+)
+from .configured_asset_filesystem_data_connector import (  # isort:skip
     ConfiguredAssetFilesystemDataConnector,
 )
-from .configured_asset_gcs_data_connector import ConfiguredAssetGCSDataConnector
-from .configured_asset_s3_data_connector import ConfiguredAssetS3DataConnector
-from .configured_asset_sql_data_connector import ConfiguredAssetSqlDataConnector
-from .data_connector import DataConnector
-from .file_path_data_connector import FilePathDataConnector
-from .inferred_asset_file_path_data_connector import InferredAssetFilePathDataConnector
-from .inferred_asset_filesystem_data_connector import (
+from .inferred_asset_filesystem_data_connector import (  # isort:skip
     InferredAssetFilesystemDataConnector,
 )
-from .inferred_asset_gcs_data_connector import InferredAssetGCSDataConnector
-from .inferred_asset_s3_data_connector import InferredAssetS3DataConnector
-from .inferred_asset_sql_data_connector import InferredAssetSqlDataConnector
-from .runtime_data_connector import RuntimeDataConnector
-
+from .configured_asset_s3_data_connector import (  # isort:skip
+    ConfiguredAssetS3DataConnector,
+)
+from .inferred_asset_s3_data_connector import InferredAssetS3DataConnector  # isort:skip
 from .configured_asset_azure_data_connector import (  # isort:skip
     ConfiguredAssetAzureDataConnector,
 )
+from .inferred_asset_azure_data_connector import (  # isort:skip
+    InferredAssetAzureDataConnector,
+)
+from .configured_asset_gcs_data_connector import (  # isort:skip
+    ConfiguredAssetGCSDataConnector,
+)
+from .inferred_asset_gcs_data_connector import (  # isort:skip
+    InferredAssetGCSDataConnector,
+)
+from .configured_asset_sql_data_connector import (  # isort:skip
+    ConfiguredAssetSqlDataConnector,
+)
+from .inferred_asset_sql_data_connector import (  # isort:skip
+    InferredAssetSqlDataConnector,
+)
```

Comment:
https://here-be-pythons.readthedocs.io/en/latest/python/isort.html

Could we please use the file-level isort:skip?

Reply:
@cdkini Done!!!  Learn something new every day -- thanks!